README.md

```diff
@@ -27,7 +27,32 @@ Download the binary for your platform from the [releases](https://github.com/ran
 wget https://github.com/rancher/terraform-provider-rke/releases/download/x.x.x/terraform-provider-rke_<platform>
 ```
 
-Rename the binary to to remove the platform ending (so it looks like `terraform-provider-rke`)and move it to your terraform plugin directory (`~/terraform.d/plugins/linux_amd64`) or a custom directory. If using a custom directory use the `-plug-in=/my/custom/path` when running `terraform init` to specify the path.
+1. rename it to `terraform-provider-rke_v0.14.1`
+1. put it to `~/.terraform.d/plugins`
```

Comment:
Not always, could be `.terraform/plugins/<OS>_<ARCH>`, `~/.terraform.d/plugins/<OS>_<ARCH>` , custom directory...

Reply:
therefore I put the detail link in `line 33`

Reply:
Yes, but may be by default is better use <OS>_<ARCH>  folder

Reply:
updated thx